src/NHSD.BuyingCatalogue.Solutions.API/Controllers/Suppliers/SuppliersController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Solutions.API.ViewModels.Suppliers;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.Controllers.Suppliers
+{
+    [Route("api/v1/suppliers")]
+    [ApiController]
+    [Produces("application/json")]
+    [AllowAnonymous]
+    public sealed class SuppliersController : Controller
+    {
+        [HttpGet]
+        public ActionResult GetList(string name)
+        {
+            var result = new List<GetSuppliersNameResult>
+            {
+                new GetSuppliersNameResult() {SupplierId = "id", Name = name}
```

Comment:
The brackets can be omitted.

Reply:
This will change anyways for the implementation, so left it how it is for now